pkg/ebpftracer/module.go

```diff
@@ -110,18 +108,7 @@ func (m *module) load(cfg Config) error {
 	}
 
 	m.objects = &objs
-
-	// TODO(Kvisord): Mount cgroupv2 if not mounted.
-	cgroupPath, err := detectCgroupPath()
-	if err != nil {
-		cgroupPath = "/cgroupv2"
-		m.log.Debugf("mounting cgroupv2 to path %s", cgroupPath)
-		if err := mountCgroup2(cgroupPath); err != nil {
-			return fmt.Errorf("mounting cgroupv2: %w", err)
-		}
-	}
-	m.probes = newProbes(m.objects, cgroupPath)
-
+	m.probes = newProbes(m.objects, cfg.HostCgroupsPath)
```

Comment:
Use the cgroups path directly from the [`--host-cgroups` flag](https://github.com/castai/kvisor/blob/0ae171db3209b34c503d4b4da0281a50b0aa4224/cmd/agent/daemon/daemon.go#L48).

Reply:
Hmm I would prefer having it as part of the `CgroupClient`. That way, most cgroup related operations would be encapsulated by it.

Reply:
Done!